spec/support/capybara.rb

```diff
@@ -20,6 +20,8 @@
 RSpec.configure do |config|
   config.before(:each, type: :system) do
     driven_by(:selenium, using: ENV.fetch("CAPYBARA_DRIVER", "headless_chrome").to_sym,
-                         screen_size: [1920, 1080])
+                         screen_size: [1920, 1080]) do |capabilities|
+      capabilities.add_argument("--no-sandbox") if ENV["CAPYBARA_NO_SANDBOX"].present?
```

Comment:
what does this do?

Reply:
I had to add it because otherwise chromium will fail to run inside the dev container.
Thus it's only enabled in the `Dockerfile` and can otherwise be ignored.